pgx-examples/aggregate/.cargo/pgx-linker-script.sh

```diff
@@ -2,7 +2,17 @@
 # Auto-generated by pgx. You may edit this, or delete it to have a new one created.
 
 if [[ $CARGO_BIN_NAME == "sql-generator" ]]; then
-    gcc -Wl,-undefined,dynamic_lookup,-dynamic-list=$CARGO_MANIFEST_DIR/.cargo/pgx-dynamic-list.txt $@
-else 
+    if [[ $TARGET == "x86_64-apple-darwin" ]] || [[ $TARGET == "aarch64-apple-darwin" ]]; then
```

Comment:
Any chance we can delete these files from source control but still have things work?  Does `cargo pgx run/tinstall/text/package` create these files automatically if they don't exist?

Reply:
We discussed this previously and while it's possible I don't think it's a good idea as not all build environments have a mutable source tree, and  it feels much too ~magic~ for my comfort.

Reply:
Yes. Right. Okay. Leave it as-is.